packages/cap-config-writer/test/unit/cap-writer/updates.test.ts

```diff
@@ -0,0 +1,135 @@
+import { applyCAPUpdates } from '../../../src/cap-writer/updates';
+import type { CapServiceCdsInfo, CapProjectSettings } from '../../../src/index';
+import type { CapRuntime } from '@sap-ux/odata-service-inquirer';
+import { join } from 'path';
+import { updateRootPackageJson, updateAppPackageJson } from '../../../src/cap-writer/package-json';
+import { updateTsConfig, updateStaticLocationsInApplicationYaml } from '../../../src/cap-writer/tsconfig-and-yaml';
+import { updatePomXml } from '../../../src/cap-writer/pom-xml';
+
+// Mocks
+jest.mock('../../../src/cap-writer/package-json', () => ({
```

Comment:
> Should we be mocking these? We are not testing the outputs here at all. It would be good to have at least one test that actually makes sure the outputs from calling `applyCapUpdates` is correct, and that the interaction of each function on the same files is not causing issues (there is no coverage for that currently. )

Removed the mocks and tested applyCapUpdates for outputs based on options passed